tests/test_ble_interface_advanced.py

```
@@ -830,13 +842,13 @@ def test_ble_client_async_timeout_maps_to_ble_error(
     # BLEClient and BLEInterface already imported at top as ble_mod.BLEClient, ble_mod.BLEInterface
 
-    client = ble_mod.BLEClient()  # address=None keeps underlying bleak client unset
+    client = BLEClient()  # address=None keeps underlying bleak client unset
     fake_future = _make_fake_future(FutureTimeoutError())
     monkeypatch.setattr(client, "_async_run", _bind_coro_to_future(fake_future))
 
-    async def _test_coro():
+    async def _test_coro() -> None:
         """Run a no-op coroutine used for tests."""
-        return None
+        pass
```

Ruff (0.15.4)

[warning] Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)